### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**
* Move Advanced Config section to a separate file
* Add build & release badges
* Simplify Getting Started example

### CODE REVIEW

File: docs/advanced-config.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+| `otel.instrumentation.methods.include`                           | `OTEL_INSTRUMENTATION_METHODS_INCLUDE`                           | unset         | Same as adding `@WithSpan` annotation functionality for the target method string. <details><summary>Format</summary>`my.package.MyClass1[method1,method2];my.package.MyClass2[method3]`</details>
+| `otel.instrumentation.opentelemetry-annotations.exclude-methods` | `OTEL_INSTRUMENTATION_OPENTELEMETRY_ANNOTATIONS_EXCLUDE_METHODS` | unset         | Suppress `@WithSpan` instrumentation for specific methods. <details><summary>Format</summary>`my.package.MyClass1[method1,method2];my.package.MyClass2[method3]`</details>
+
+## Java agent configuration
+
+| System property          | Environment variable     | Default value  | Purpose                                          |
+| ------------------------ | ------------------------ | -------------- | -------------------------------------------------|
+| `otel.javaagent.enabled` | `OTEL_JAVAAGENT_ENABLED` | `true`         | Globally enables javaagent auto-instrumentation. |
+
+You can find more Java agent configuration
```
Comment (kubawach) [inline docs/advanced-config.md:44]: This perhaps should get own section (# Generic OpenTelemetry instrumentation configuration).

File: README.md [hunk 2]
```diff
@@ -59,16 +76,22 @@ To see the Java Agent in action with sample applications, see our
 ### Basic Configuration
 
 The service name resource attribute is the only configuration option
-that typically needs to be specified. You can set it by adding a `service.name`
+that is required by the javaagent. You can set it by adding a `service.name`
```
Comment (kubawach) [inline README.md:79]: I'm not sure about the wording - it sounds as if the agent don't run without the property (which is not true). I'd change to "should be specified".

File: README.md [hunk 3]
```diff
@@ -18,33 +31,37 @@ This Splunk distribution comes with the following defaults:
...
-    -o splunk-otel-javaagent.jar
-$ java -javaagent:./splunk-otel-javaagent.jar -Dotel.resource.attributes=service.name=my-java-app \
-    -jar target/java-agent-example-1.0-SNAPSHOT-shaded.jar https://google.com
+java -javaagent:./splunk-otel-javaagent.jar \
+     -Dotel.resource.attributes=service.name=my-java-app \
+     -jar myapp.jar https://google.com
 ```
 
-> The `-javaagent` needs to be run before the `-jar` file,
+> The `-javaagent` needs to appear before the `-jar` file,
```
Comment (kubawach) [inline README.md:64]: Nice

**Review by kubawach — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (mateuszrzeszutek) [reply README.md:79]: Eh, true -- it should be required though 😛

**Review by jkwatson — APPROVED** [review]
Looks good!

**Pull request merged**